ui/apps/platform/src/Containers/SystemConfig/Form/SystemConfigForm.tsx

```diff
@@ -72,6 +77,17 @@ export type SystemConfigFormProps = {
     setIsNotEditing: () => void;
 };
 
+const validationSchema = yup.object().shape({
+    privateConfig: yup.object().shape({
+        reportRetentionConfig: yup.object().shape({
+            downloadableReportGlobalRetentionBytes: yup
+                .number()
+                .min(convertBetweenBytesAndMB(50, 'MB'), 'The number must be at least 50 MB')
+                .required(),
```

Comment:
Brad, does this property have a default value like other properties have `0` for never delete?

That is, what is the default behavior in backend if you do not set the value here?

The integration tests fail because the **Save** button is disabled, probably because of `required()` here.

Reply:
It looks like default value changed from original 5MB to 500MB yesterday in #7395
https://github.com/stackrox/stackrox/blob/master/central/config/datastore/singleton.go#L36-L37

Reply:
Ah I was waiting for that change to happen and was expecting it to come from Charmik. I rebased to get those changes. 

I'll check if 0 is allowed but my intuition says no.